Global-API/health.py

```diff
@@ -0,0 +1,26 @@
+from fastapi import FastAPI
+from sqlalchemy import create_engine, exc
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+app = FastAPI()
+
+DATABASE_URL = "TBD"
+
+# Create a SQLAlchemy engine for database connection
+engine = create_engine(DATABASE_URL)
+
+# Create a session maker for SQLAlchemy
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+# Define a base class for SQLAlchemy models
+Base = declarative_base()
+
+@app.get("/health")
+def health_check():
+    try:
+        engine.connnect()
```

Comment:
Typo in method name.
Usually I think you would have a persistent connection to the database that you can just check if it still exists. This might be different for SQLalchemy/ Fast API though, so let's follow the best practices for that.